src/themes/dspace/app/submission/sections/upload/file/section-upload-file.component.scss

```diff
@@ -0,0 +1,16 @@
+.action-list {
+    * {
+      height: 100%;  
+    }
+    .fa {
+        font-size: 1.2rem;
+    }
+}
+
+.action-btn {
+    font-size: 1.2rem;
+}
+
+.flex-grow {
+    flex-grow: 10;
+}
```

Comment:
1) no need to specify "10", only "1" should be necessary if other flexbox elements don't specify any "flex-grow".
2) you don't need to specify a class, you could use the `flex-grow-1` provided by bootstrap